stories/dooboo-ui/Checkbox.stories.tsx

```diff
@@ -27,6 +27,49 @@ const Selected = styled.Text`
   font-weight: bold;
 `;
 
+const defaultCheckboxData: any[] = [
```

Comment:
It can be `const defaultData:  CheeckboxItem[]` if you defined `CheckboxItem` interface.